code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -68,6 +68,28 @@
 	poison_type = /singleton/reagent/soporific
 	var/fed = 0
 
+/mob/living/simple_animal/hostile/giant_spider/nurse/servant
+	name = "greimorian servant"
+	desc = "A greimorian with a startling intelligence to its bulbous yellow eyes. Its needle-like mandibles look like they could easily punch through armor - or flesh."
+	icon_state = "greimorian_servant"
+	icon_living = "greimorian_servant"
+	icon_dead = "greimorian_servant_dead"
+	blood_amount = 150
+	maxHealth = 200
+	health = 200
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	armor_penetration = 30
+	poison_per_bite = 10
+	speed = -3
```

Comment:
seems excessively fast for a mob this good

Reply:
Nerfed to -2